app-refresh/src/main/java/com/androidessence/cashcaretaker/addaccount/AddAccountPresenterImpl.kt

```diff
@@ -31,7 +31,7 @@ class AddAccountPresenterImpl(private var addAccountView: AddAccountView?, priva
             return
         }
 
-        addAccountView?.showProgress()
+        controller?.showProgress()
```

Comment:
This should have set the process to loading. In fact, the controllers no longer need the progress methods.

Reply:
Scratch that, this is more complicated I'm gonna make a separate ticket for insert flows.